### PR TITLE
New model release

### PR DESCRIPTION
**Pull request by xkvu**
- Remove some logic using AsyncTask (deprecated)
- Separate the predicting logic to a different class AudioProcessors
- Wearable.SoundRecorder is now sending 5360 * 2 bytes to phone each time instead of minBufferSize

### CODE REVIEW

**Comment by xkvu** [comment]
@QuanGary One big thing to note is that I changed the size of each packet sent from watch to phone. Before, it's just == `AudioRecorder.getMinBuffer()` (for the watch I used, it was ~ 640bytes). I changed it to `5360 * 2` bytes so that when the phone receives the message, it will immediately have a big enough array of shorts (size of 5360) to make a prediction immediately. Note that I also add the recording time at the beginning of the message packet (I think this might be helpful later if we want to pinpoint the exact time when the audio is recorded).

Long story short, please check if there is some latency issue going on and let me know. Thanks 🥇
